epsagon/trace.py

```diff
@@ -717,9 +720,9 @@ def add_label(self, key, value):
                 self.add_label('{}.{}'.format(key, dict_key), dict_value)
             return
 
-        # Convert numbers to string.
-        if isinstance(value, (int, float)):
-            value = str(value)
+        # Convert boolean to string.
+        if isinstance(value, bool):
```

Comment:
`bool` should be saved as is (it's a valid JSON type).
Note that it's a subtype of `int`.

Reply:
👍